lib/SoyCompiler.js

```diff
@@ -299,6 +303,25 @@ SoyCompiler.prototype._compileTemplateFilesAndEmit = function (inputDir, files,
   })
 
   cp.on('exit', onExit)
+
+  return deferred.promise.then(function () {
+    var vmTypes = [DEFAULT_VM_CONTEXT]
+    if (options.locales && options.locales.length > 0) {
+      vmTypes = options.locales.concat()
```

Comment:
If this is just to clone the array, maybe use `.slice()` or `Array.from()` to be more clear that this isn't a bug?